ClientLibrary/FaceServiceClient.cs

```diff
@@ -571,14 +571,31 @@ public static string GetAttributeString(IEnumerable<FaceAttributeType> types)
         /// <returns>
         /// The person entity array.
         /// </returns>
+        [Obsolete("use ListPersonsAsync instead")]
         public async Task<Person[]> GetPersonsAsync(string personGroupId)
         {
+            return await ListPersonsAsync(personGroupId);
+        }
+
+        /// <summary>
+        /// List the top persons whose Id is larger than "start" inside a person group asynchronously.
+        /// </summary>
+        /// <param name="personGroupId">The person group id.</param>
+        /// <param name="start">person Id bar. List the persons whose Id is larger than "start".</param>
+        /// <param name="top">the number of persons to list.</param>>
+        /// <returns>
+        /// The person entity array.
+        /// </returns>
+        public async Task<Person[]> ListPersonsAsync(string personGroupId, string start = "", int top = 1000)
```

Comment:
missing the function with only "top" parameter.

 public async Task<Person[]> ListPersonsAsync(string personGroupId, int top = 1000) 
{
      return await ListPersonsAsync(personGroupId, "", top); 
}